iceoryx_hoofs/include/iceoryx_hoofs/log/building_blocks/console_logger.hpp

```diff
@@ -107,19 +117,21 @@ class ConsoleLogger
         static constexpr uint32_t BUFFER_SIZE{1024};
         static constexpr uint32_t NULL_TERMINATED_BUFFER_SIZE{BUFFER_SIZE + 1};
 
-        // NOLINTJUSTIFICATION safe access is guaranteed since the char array is wrapped inside the class
+        // AXIVION Next Construct AutosarC++19_03-A3.9.1 : Not used as an integer but as actual character
+        // AXIVION Next Construct AutosarC++19_03-A18.1.1 : This is a low-level component with minimal dependencies.
+        // Safe access is guaranteed since the char array is wrapped inside the class
         // NOLINTNEXTLINE(hicpp-avoid-c-arrays, cppcoreguidelines-avoid-c-arrays)
         char buffer[NULL_TERMINATED_BUFFER_SIZE];
         uint32_t bufferWriteIndex; // initialized in corresponding cpp file
         /// @todo iox-#1345 add thread local storage with thread id and print it in the log messages
     };
 
-    static ThreadLocalData& getThreadLocalData();
+    static ThreadLocalData& getThreadLocalData() noexcept;
 
   private:
     // NOLINTJUSTIFICATION needed for the functionality and a private member of the class
     // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-    static std::atomic<LogLevel> m_activeLogLevel; // initialized in corresponding cpp file
+    static std::atomic<LogLevel> s_activeLogLevel; // initialized in corresponding cpp file
```

Comment:
Why not `sm_`? :sweat_smile: Shall we document that as new convention somewhere?

Reply:
Hahaha, it is already used in some places. Not sure if this is required since it is a implementation detail